src/code/qualification/getSimFile.m

```diff
@@ -25,7 +25,7 @@
             strcmp(SimulationMapping(i).Simulation, Simulation)
         
         path = SimulationMapping(i).Path;
-        csvSimFile = fullfile(path, [Simulation '-Results.csv']);
-        xmlSimFile = fullfile(path, [Simulation '.xml']);
+        csvSimFile = fullfile(pwd, path, [Simulation '-Results.csv']);
+        xmlSimFile = fullfile(pwd, path, [Simulation '.xml']);

```

Comment:
`pwd `is the current folder. When the code is executed: is this always the folder of the _RE_configuration_plan_?

Reply:
So far it is. But it may be better if I allow the Qualification Workflow to be run from any directory.
I will add an issue for this and make the updates.